### PR TITLE
Epmlabsbrn 171

### PR DESCRIPTION
**Pull request by iki6**
[EPMLABSBRN-171](https://jira.epam.com/jira/browse/EPMLABSBRN-171)

**Description**: Reorganize series/exercises/tasks position

### CODE REVIEW

**Comment by iki6** [comment]
merged 187 and 188 into the general task branch(171)
set `WIP` till @LisaTomaeva checks

**Comment by H1D** [comment]
Ahh! Finally not a @dependabot PR 
image

File: frontend/app/components/series-navigation/template.hbs [hunk 1]
```diff
@@ -1,36 +1,30 @@
...
-              disabled=(not exercise.canInteract)
-            }}
-              {{capitalize exercise.name}}
-            {{/link-to}}
-          {{/if}}
-        {{/let}}
-      </li>
+          />
+          <div>
+            Level{{exercise.level}}
```
Comment (H1D) [inline frontend/app/components/series-navigation/template.hbs]: why english suddenly?

File: frontend/app/components/start-task-button/template.hbs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<button data-test-start-task-button onclick={{action this.startTask}}>
```
Comment (H1D) [inline frontend/app/components/start-task-button/template.hbs]: this button is pure awesomeness. But it should have `:hover` and `:focus` states as well as `aria-label=...`
image

**Comment by H1D** [comment]
@iki6 
image

also this button should have some kind of a hint what is it about. Maybe icon, text.

Current variant has both text and 🔈 icon
image

**Pull request merged**